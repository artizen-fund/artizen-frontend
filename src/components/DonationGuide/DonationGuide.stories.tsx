import DonationGuide from './'

export default {
  title: 'components/DonationGuide',
  component: DonationGuide,
  argTypes: {},
}

export const DonationGuideComponent = (props: any) => <DonationGuide {...props} />
