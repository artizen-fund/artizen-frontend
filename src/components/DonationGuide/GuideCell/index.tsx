import React from 'react'
import styled from 'styled-components'
import { rgba, assetPath } from '@lib'
import { palette, breakpoint } from '@theme'
import { IDonationGuideCell, DonationGuideCoordinate } from '@copy/home'

const GuideCell = ({ columns, height, items, step }: IDonationGuideCell & { step: number }) => (
  <Wrapper {...{ columns, height }}>
    {items.map((item, i) =>
      item.type === 'copy' ? (
        <CellItem key={`guide-cell-${step}-${i}`} coordinates={item.coordinates} type={item.type}>
          {i === 0 && <StepNumber>{step + 1}</StepNumber>}
          <p>{item.copy}</p>
        </CellItem>
      ) : item.type === 'image' ? (
        <CellItem key={`guide-cell-${step}-${i}`} coordinates={item.coordinates} type={item.type}>
          <img src={`${assetPath(`/assets/illustrations/guide/${item.image}`)}?fm=webp`} />
        </CellItem>
      ) : (
        <>{/* todo: Lottie animation support */}</>
      ),
    )}
  </Wrapper>
)

const StepNumber = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  min-width: 32px;
  min-height: 32px;

  background-image: url(${assetPath('/assets/step-background.png')});
  background-repeat: no-repeat;
  background-size: contain;

  font-family: 'roc-grotesk';
  font-style: normal;
  font-weight: 700;
  font-size: 16px;
  letter-spacing: 0.5px;
  color: ${rgba(palette.night)};
`

const getCoordinate = (n: number, axis: 'x' | 'y') => {
  if (axis === 'x' && n < 0) {
    return `right: ${n * -1}%`
  }
  if (axis === 'x') {
    return `left: ${n}%`
  }
  if (n < 0) {
    return `bottom: ${n * -1}%`
  }
  return `top: ${n}%`
}

const getWidth = (cellType: string, n?: number) => {
  if (!n) {
    return '100%'
  }
  if (cellType === 'copy') {
    return `${n}px`
  }
  return `${n}%`
}

const CellItem = styled.div<{ coordinates: DonationGuideCoordinate; type: string }>`
  position: absolute;
  z-index: ${props => (props.type === 'copy' ? 1 : 2)};

  @media only screen and (max-width: ${breakpoint.phablet - 1}px) {
    ${props => getCoordinate(props.coordinates.mobile.x, 'x')};
    ${props => getCoordinate(props.coordinates.mobile.y, 'y')};
    width: ${props => getWidth(props.type, props.coordinates.mobile.w)};
  }
  @media only screen and (min-width: ${breakpoint.phablet}px) and (max-width: ${breakpoint.tablet - 1}px) {
    ${props => getCoordinate(props.coordinates.phablet.x, 'x')};
    ${props => getCoordinate(props.coordinates.phablet.y, 'y')};
    width: ${props => getWidth(props.type, props.coordinates.phablet.w)};
  }
  @media only screen and (min-width: ${breakpoint.tablet}px) and (max-width: ${breakpoint.laptop - 1}px) {
    ${props => getCoordinate(props.coordinates.tablet.x, 'x')};
    ${props => getCoordinate(props.coordinates.tablet.y, 'y')};
    width: ${props => getWidth(props.type, props.coordinates.tablet.w)};
  }
  @media only screen and (min-width: ${breakpoint.laptop}px) and (max-width: ${breakpoint.desktop - 1}px) {
    ${props => getCoordinate(props.coordinates.laptop.x, 'x')};
    ${props => getCoordinate(props.coordinates.laptop.y, 'y')};
    width: ${props => getWidth(props.type, props.coordinates.laptop.w)};
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    ${props => getCoordinate(props.coordinates.desktop.x, 'x')};
    ${props => getCoordinate(props.coordinates.desktop.y, 'y')};
    width: ${props => getWidth(props.type, props.coordinates.desktop.w)};
  }

  img {
    object-fit: contain;
    max-width: 100%;
  }

  ${props =>
    props.type === 'copy' &&
    `
    display: flex;
    flex-direction: row;
    align-items: flex-start;
    gap: 5px;
  `}

  p {
    display: flex;

    background: #f1f3ee;
    border-radius: 16px;
    padding: 12px;

    font-family: 'roc-grotesk';
    font-size: 13px;
    font-style: normal;
    font-weight: 600;
    line-height: 120%;
    letter-spacing: -0.4px;
    @media only screen and (min-width: ${breakpoint.phablet}px) {
      padding: 14px;
      font-size: 13px;
    }
    @media only screen and (min-width: ${breakpoint.tablet}px) {
      padding: 14px;
      font-size: 14px;
    }
    @media only screen and (min-width: ${breakpoint.laptop}px) {
      padding: 16px;
      font-size: 14px;
    }
    @media only screen and (min-width: ${breakpoint.desktop}px) {
      padding: 32px;
      font-size: 20px;
    }
  }

  color: ${rgba(palette.night)} !important;
`

/* TODO: note that these height calculations are off
 * actual calculation is (100vw - margin of popup - padding of popup) * height multiplier
 * I'll come around to this.
 */

const Wrapper = styled.div<Pick<IDonationGuideCell, 'columns' | 'height'>>`
  position: relative;
  grid-column-end: span ${props => props.columns.mobile};
  height: calc((100vw - 40px) * ${props => props.height.mobile});

  @media only screen and (min-width: ${breakpoint.phablet}px) {
    grid-column-end: span ${props => props.columns.phablet};
    height: calc((100vw - 40px) * ${props => props.height.phablet});
  }
  @media only screen and (min-width: ${breakpoint.tablet}px) {
    grid-column-end: span ${props => props.columns.tablet};
    height: ${props => props.height.tablet}px;
  }
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    grid-column-end: span ${props => props.columns.laptop};
    height: ${props => props.height.laptop}px;
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    grid-column-end: span ${props => props.columns.desktop};
    height: ${props => props.height.desktop}px;
  }
`

export default GuideCell
