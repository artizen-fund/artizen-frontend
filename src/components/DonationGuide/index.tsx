import { useContext } from 'react'
import styled from 'styled-components'
import { CloseButton } from '@components'
import { rgba, DonationContext } from '@lib'
import { palette, typography, breakpoint } from '@theme'
import { donationGuideMap } from '@copy/home'
import GuideCell from './GuideCell'

const DonationGuide = () => {
  const { visibleModal, toggleModal } = useContext(DonationContext)
  const visible = visibleModal === 'donationGuide'
  return (
    <>
      <Wrapper {...{ visible }}>
        <Headline>
          How to donate to grants
          <br />
          using web3 — a guide
        </Headline>

        {donationGuideMap.map((item, i) => (
          <GuideCell key={`guidecell-${i}`} {...item} step={i} />
        ))}
      </Wrapper>
      <StyledCloseButton glyph="cross" level={1} onClick={() => toggleModal?.()} {...{ visible }} />
    </>
  )
}

type VisibilityParam = {
  visible: boolean
}

const Wrapper = styled.div<VisibilityParam>`
  position: fixed;
  z-index: 102;
  left: 20px;
  top: 75px;
  width: calc(100vw - 40px);
  height: calc(100vh - 75px);
  @media only screen and (min-width: ${breakpoint.tablet}px) {
    left: 50px;
    top: 125px;
    width: calc(100vw - 100px);
    height: calc(100vh - 125px);
  }
  overflow-y: scroll;

  display: grid;
  grid-template-rows: auto;
  grid-template-columns: repeat(4, 1fr);
  grid-column-gap: 12px;
  grid-row-gap: 32px;
  padding: 24px;

  font-size: 13px;
  line-height: 120%;
  letter-spacing: -0.4px;

  @media only screen and (min-width: ${breakpoint.phablet}px) {
    grid-column-gap: 16px;
    grid-row-gap: 48px;
    padding: 40px;
  }

  @media only screen and (min-width: 780px) {
    width: 700px;
    left: calc((100vw - 700px) / 2);
    grid-template-columns: repeat(12, 1fr);
    font-size: 14px;
  }

  @media only screen and (min-width: 1044px) {
    width: 1024px;
    left: calc((100vw - 1024px) / 2);
    grid-column-gap: 20px;
    grid-row-gap: 72px;
  }

  @media only screen and (min-width: 1720px) {
    width: 1680px;
    left: calc((100vw - 1680px) / 2);
    grid-column-gap: 25px;
    grid-row-gap: 80px;
    padding: 80px;
    font-size: 20px;
  }

  background: white;

  transition: opacity 0.25s ease-in-out, transform 0.3s ease-in-out;
  opacity: ${props => (props.visible ? 1 : 0)};
  transform: translateY(${props => (props.visible ? 0 : '100px')};);
  pointer-events: ${props => (props.visible ? 'all' : 'none')};
`

const Headline = styled.h1`
  position: absolute;
  z-index: 3;
  top: 24px;
  left: 24px;

  @media only screen and (min-width: ${breakpoint.phablet}px) {
    top: 40px;
    left: 40px;
  }

  @media only screen and (min-width: ${breakpoint.desktop}px) {
    top: 80px;
    left: 80px;
  }

  ${typography.title.l1}
  color: ${rgba(palette.black)};
`

const StyledCloseButton = styled(props => <CloseButton {...props} />)`
  top: 60px;
  right: 0px;
  @media only screen and (min-width: ${breakpoint.phablet}px) {
    top: 100px;
    right: 25px;
  }
`

export default DonationGuide
