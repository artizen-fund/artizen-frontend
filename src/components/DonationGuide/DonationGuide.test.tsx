import renderer from 'react-test-renderer'
import DonationGuide from './'

describe('DonationGuide', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<DonationGuide />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
