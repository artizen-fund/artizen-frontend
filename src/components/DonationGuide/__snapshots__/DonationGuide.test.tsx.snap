// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonationGuide renders correctly 1`] = `
Array [
  <div
    className="sc-213fa555-0 dZEntA"
  >
    <h1
      className="sc-213fa555-1 bZKPya"
    >
      How to donate to grants
      <br />
      using web3 — a guide
    </h1>
    <div
      className="sc-446f2393-2 csvqBM"
      height={
        Object {
          "desktop": 0.48,
          "laptop": 0.48,
          "mobile": 1.4,
          "phablet": 0.7,
          "tablet": 0.5,
        }
      }
    >
      <div
        className="sc-446f2393-1 jAqPyk"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          1
        </div>
        <p>
          First, if you’re stressed about web3, don’t. We got you. It’s easier than you think.
        </p>
      </div>
      <div
        className="sc-446f2393-1 lFWcA"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/01.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 feABgX"
      height={
        Object {
          "desktop": 0.25,
          "laptop": 0.25,
          "mobile": 0.9,
          "phablet": 0.45,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 iVjfef"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          2
        </div>
        <p>
          The grants are funded using crypto. But you can donate using two payment methods:
        </p>
      </div>
      <div
        className="sc-446f2393-1 jQjSjI"
        type="copy"
      >
        <p>
          Your credit card (We’ll take care of turning your money to crypto, no worries) and Crypto (Polygon or Ethereum).
        </p>
      </div>
      <div
        className="sc-446f2393-1 cUVFSU"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/02a.png"
        />
      </div>
      <div
        className="sc-446f2393-1 ftwlKd"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/02b.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 giJrSr"
      height={
        Object {
          "desktop": 0.32,
          "laptop": 0.32,
          "mobile": 0.7,
          "phablet": 0.45,
          "tablet": 0.32,
        }
      }
    >
      <div
        className="sc-446f2393-1 eUqgLc"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          3
        </div>
        <p>
          Let’s take a look at donating with credit card first!
        </p>
      </div>
      <div
        className="sc-446f2393-1 dwxenl"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/03.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 iyevEe"
      height={
        Object {
          "desktop": 0.32,
          "laptop": 0.34,
          "mobile": 0.95,
          "phablet": 0.45,
          "tablet": 0.32,
        }
      }
    >
      <div
        className="sc-446f2393-1 hByJwU"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          4
        </div>
        <p>
          Select your donation amount. 100% goes to the grant but expect payment provider fees on top (we don’t charge any).
        </p>
      </div>
      <div
        className="sc-446f2393-1 gYhLOH"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/04.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 jNPiaj"
      height={
        Object {
          "desktop": 0.32,
          "laptop": 0.34,
          "mobile": 1,
          "phablet": 0.45,
          "tablet": 0.34,
        }
      }
    >
      <div
        className="sc-446f2393-1 hQpMkK"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          5
        </div>
        <p>
          Sign in (the process is exactly the same whether you’ve already donated or not.) There’s no password to set up and remember (phew!).
        </p>
      </div>
      <div
        className="sc-446f2393-1 gHkoWV"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/05.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 kIQqJP"
      height={
        Object {
          "desktop": 0.45,
          "laptop": 0.38,
          "mobile": 1,
          "phablet": 0.45,
          "tablet": 0.34,
        }
      }
    >
      <div
        className="sc-446f2393-1 elHfXM"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          6
        </div>
        <p>
          You’ll get a confirmation email or text with a magic link (super secure!). Click on it.
        </p>
      </div>
      <div
        className="sc-446f2393-1 gOpMcU"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/06.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 gSUUtU"
      height={
        Object {
          "desktop": 0.45,
          "laptop": 0.35,
          "mobile": 1,
          "phablet": 0.6,
          "tablet": 0.45,
        }
      }
    >
      <div
        className="sc-446f2393-1 eVGWEt"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          7
        </div>
        <p>
          We’ll automatically create your Artizen wallet for you. So you don’t have to get an external one. (But you can connect another wallet if you have one.)
        </p>
      </div>
      <div
        className="sc-446f2393-1 jMDYKX"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/07.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 iWivdU"
      height={
        Object {
          "desktop": 0.368,
          "laptop": 0.38,
          "mobile": 1,
          "phablet": 0.45,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 gNCHmk"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          8
        </div>
        <p>
          We’ll convert your donation amount to USDC and transfer it to your wallet.
        </p>
      </div>
      <div
        className="sc-446f2393-1 bzVwfn"
        type="copy"
      >
        <p>
          This is where things might get a little unexpected. The transaction’s not instant. It might take anywhere from 10 minutes up to 2 hours.
        </p>
      </div>
      <div
        className="sc-446f2393-1 fUgCVu"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/08.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 hriDRn"
      height={
        Object {
          "desktop": 0.368,
          "laptop": 0.38,
          "mobile": 1,
          "phablet": 0.55,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 iDYbSP"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          9
        </div>
        <p>
          THIS IS FINE.
        </p>
      </div>
      <div
        className="sc-446f2393-1 dgYpkX"
        type="copy"
      >
        <p>
          No, really, it’s fine. It’s how crypto works.
        </p>
      </div>
      <div
        className="sc-446f2393-1 eosUEa"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/09.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 blBSuE"
      height={
        Object {
          "desktop": 0.368,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.58,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 hjINtF"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          10
        </div>
        <p>
          You can close the browser window – we’ll send you an email when the donation’s ready to transfer to the Fund.
        </p>
      </div>
      <div
        className="sc-446f2393-1 eGsSnG"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/10.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 hKXODD"
      height={
        Object {
          "desktop": 0.368,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.56,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 feRgCi"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          11
        </div>
        <p>
          Already starting with crypto? You can donate using USDC on Polygon or Ethereum.
        </p>
      </div>
      <div
        className="sc-446f2393-1 bDMYvU"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/11.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 hGwWys"
      height={
        Object {
          "desktop": 0.368,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.62,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 fKqlNv"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          12
        </div>
        <p>
          In that case, sign in and connect your existing wallet to move your coins to your new Artizen wallet.
        </p>
      </div>
      <div
        className="sc-446f2393-1 lkxiWh"
        type="copy"
      >
        <p>
          (Psst! You can save your wallet connection to not have to go through the process next time.)
        </p>
      </div>
      <div
        className="sc-446f2393-1 hEXoVx"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/12.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 kTeKye"
      height={
        Object {
          "desktop": 0.368,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.61,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 kdghYo"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          13
        </div>
        <p>
          Then carefully follow the instructions from your wallet provider.
        </p>
      </div>
      <div
        className="sc-446f2393-1 gWxrdT"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/13.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 kbNBYM"
      height={
        Object {
          "desktop": 0.475,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.6,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 cVFODP"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          14
        </div>
        <p>
          After that, a bit of crypto magic like swapping and bridging will happen in the background so that your donation can safely make its way to the Artizen Fund.
        </p>
      </div>
      <div
        className="sc-446f2393-1 biFjZe"
        type="copy"
      >
        <p>
          Don’t forget that this might take some time.
        </p>
      </div>
      <div
        className="sc-446f2393-1 dqWJBa"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/14.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 kbNBYM"
      height={
        Object {
          "desktop": 0.475,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.6,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 jtayKF"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          15
        </div>
        <p>
          Ta-da! The donation’s on its way to the Artizen Fund. A receipt will land in your email inbox
        </p>
      </div>
      <div
        className="sc-446f2393-1 cHmVFh"
        type="copy"
      >
        <p>
          (This may take another few hours, though. You know the drill by now!)
        </p>
      </div>
      <div
        className="sc-446f2393-1 bJrEuM"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/15.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 gPVTwU"
      height={
        Object {
          "desktop": 0.38,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.55,
          "tablet": 0.4,
        }
      }
    >
      <div
        className="sc-446f2393-1 itiGhG"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          16
        </div>
        <p>
          And your name will land on the leaderboard. While you can enjoy all the perks that come along with it.
        </p>
      </div>
      <div
        className="sc-446f2393-1 jYEmFe"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/16.png"
        />
      </div>
    </div>
    <div
      className="sc-446f2393-2 fIdZCY"
      height={
        Object {
          "desktop": 0.38,
          "laptop": 0.36,
          "mobile": 1,
          "phablet": 0.53,
          "tablet": 0.45,
        }
      }
    >
      <div
        className="sc-446f2393-1 jqwVIX"
        type="copy"
      >
        <div
          className="sc-446f2393-0 jRxPmW"
        >
          17
        </div>
        <p>
          And the satisfaction of supporting a cause that matters.
        </p>
      </div>
      <div
        className="sc-446f2393-1 husREw"
        type="copy"
      >
        <p>
          And of making the world a slightly better place.
        </p>
      </div>
      <div
        className="sc-446f2393-1 kSCHiq"
        type="image"
      >
        <img
          src="/assets/illustrations/guide/17.png"
        />
      </div>
    </div>
  </div>,
  <div
    className="sc-189919d9-2 kkJcfa sc-213fa555-2 jkrlb"
    onClick={[Function]}
  >
    <div
      className="sc-189919d9-0 czelmB"
      color="white"
    >
      <div
        className="sc-149d3190-0 euiYxq"
        color="white"
      />
    </div>
  </div>,
]
`;
